modules/dxp/apps/commerce/commerce-shop-by-diagram/commerce-shop-by-diagram-web/src/main/resources/META-INF/resources/js/Diagram.js

```diff
@@ -10,6 +10,7 @@
  */
 
 import {ClayIconSpriteContext} from '@clayui/icon';
+import {fetch} from 'frontend-js-web';
```

Comment:
The `fetch` imported here is unused. This is why CI format test is failing. Please make sure to run `gradle formatSource` before opening PR, it will report error on this.

Reply:
removed this